exchange/src/shared/messages_exchange_to_wrapper.hpp

```diff
@@ -115,3 +124,11 @@ struct glz::meta<nutc::messages::start_time> {
     static constexpr auto value = // NOLINT
         object(&t::start_time_ns);
 };
+
+/// \cond
+template <>
+struct glz::meta<nutc::messages::algorithm_content> {
```

Comment:
we should clean this and start_message up

this is good for now (and improvement is outside the scope of this PR)